NGUIButtonInsetsExample/ViewController.swift

```diff
@@ -274,44 +289,104 @@ final class ButtonView: UIView {
     }
 }
 
+// MARK:  -
+// MARK: VC -
+
 class ViewController: UIViewController {
-    private let stackView = UIStackView()
-    private let buttonView = ButtonView()
-    private let resetButton = UIButton()
+    
+    // MARK: UI Properties -
+    
+    private lazy var buttonsView: ButtonsView = {
+        let bsv = ButtonsView()
+        bsv.translatesAutoresizingMaskIntoConstraints = false
+        bsv.accessibilityLabel = "ButtonsView"
+        return bsv
+    }()
+    
+    private lazy var resetButton: UIButton = {
+        let rsb = UIButton()
+        rsb.translatesAutoresizingMaskIntoConstraints = false
+        rsb.setTitle("Reset", for: .normal)
+        rsb.setTitleColor(.blue, for: .normal)
+        rsb.addTarget(self, action: #selector(resetTapped), for: .touchUpInside)
+        return rsb
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let stv = UIStackView()
+        stv.axis = .vertical
+        stv.spacing = 50
+        stv.translatesAutoresizingMaskIntoConstraints = false
+        stv.accessibilityLabel = "Main StackView"
+        stv.backgroundColor = .lightGray
+        return stv
+    }()
+    
+    private lazy var allInsetsView: AllInsetsView = {
+        let aiv = AllInsetsView(insetsDidChange: { [weak self] type, insets in
+            guard let `self` = self else { return }
+            self.buttonsView.set(insets: insets, type: type)
+        })
+        aiv.translatesAutoresizingMaskIntoConstraints = false
+        aiv.accessibilityLabel = "AllInsetsView"
```

Comment:
To be completely honest, I'm not as familiar with iOS accessibility as I'd like to be. However, as I understand it, the only views that need accessibility labels are the ones which will be read by VoiceOver. Is that true? In this case, I think these views which are implementation details of the UI (`allInsetsView`, `stackView`, `buttonsView`) don't need accessibility labels, and in fact, having accessibility labels might make the interface more confusing to use in VoiceOver.

It's possible you might be more familiar with accessibility than I am, so please let me know if I'm wrong, but could we remove these?

Reply:
I added accessibility labels solely for the purpose of debugging. I use Reveal app and it shows the views' names fro the accessibilityLabel property. 
Those `debug` names should not go into production for they don't help the user. I shall remove them.

Reply:
Nice! For what it's worth, I think Reveal will show `accessibilityIdentifier`s as well, so that could be another option for the future (accessibility identifiers don't get read during VoiceOver)